docs/guest.md

```diff
@@ -0,0 +1,40 @@
+# Tanzu Kubernetes Grid Service
+This document discusses the velero vSphere plugin installation process in a Tanzu Kubernetes Grid Service environment
+aka as Guest Cluster. For more information on Tanzu Kubernetes Grid Service refer to their [documentation](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html).
+
+## Compatibility
+| vSphere/ESXi Version       | vSphere CSI Version                                                | Kubernetes Version | Velero Version    | Velero Plugin for vSphere Version |
+|----------------------------|--------------------------------------------------------------------|--------------------|-------------------|-----------------------------------|
+| vSphere 7.0P02/ESXi 7.0P02 | Driver version is not v2.1.0, but feature is equivalent to v2.1.0. | v1.16-v1.19        | v1.5.1 and higher | v1.1.0 and higher                 |
+
+## Prerequisites
+* Prior to the installation of Velero vSphere plugin in Guest cluster it is necessary to have Velero and Velero 
+vSphere plugin installed in the Supervisor Cluster, please refer to [Supervisor Cluster Velero vSphere Plugin](docs/supervisor.md)
+installation guide.
+* DataManager needs to be installed, please refer [DataManager Documentation](docs/supervisor-datamgr.md)
+
+## Install
+Please refer [Vanilla Installation Guide](docs/vanilla.md)
```

Comment:
Should not refer to a Vanilla guide.  The install part should be in a separate doc.